.github/workflows/production.yml

```diff
@@ -52,6 +52,11 @@ jobs:
           -DskipTests=true
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: DataDog sourcemap upload
+        run: npx --package=@datadog/datadog-ci datadog-ci sourcemaps upload tools/cldr-apps/src/main/webapp/dist/ --minified-path-prefix=/cldr-apps/dist/ --release-version=r${{ github.event.inputs.git-ref }} --service=surveytool
+        env:
+          DATADOG_SITE: ${{ secrets.DATADOG_SITE }}
+          DATADOG_API_KEY: ${{ secrets.DATADOG_API_KEY }}
```

Comment:
I don't know yml well enough to know if/how the duplication of code in maven.yml and production.yml could be avoided...

For future reference I found these questions on stackoverflow, but it would take me a long time to figure out if they're helpful in this case:

https://stackoverflow.com/questions/60544181/how-to-use-snippets-in-github-action-workflow-file-to-avoid-duplicates

https://stackoverflow.com/questions/59757355/reuse-portion-of-github-action-across-jobs

Reply:
You can reuse an external action, but it's not just an #include. When you use `uses: ./.github/actions/my-action` you can't configure as much about what's inside.

That said we could have a datadog source upload action I suppose.

Reply:
I added your links to CLDR-15085